internal/internal_workflow.go

```diff
@@ -695,9 +696,6 @@ func (c *channelImpl) Close() {
 		callback.fn(nil, false)
 	}
 	// All blocked sends are going to panic
-	for _, callback := range c.blockedSends {
-		callback.fn()
```

Comment:
I don't quite understand why we want to consider the value for blocked sends consumed when closing a channel.